src/dbwrapper.h

```diff
@@ -67,10 +66,6 @@ class CDBWrapper;
  */
 namespace dbwrapper_private {
 
-/** Handle database error by throwing dbwrapper_error exception.
```

Comment:
ca6f83d88cb241786676bfb341fc27f6467d39ee: this description, although not amazing, went missing